project-fortis-pipeline/docs/user/admin.md

```diff
@@ -0,0 +1,88 @@
+# Admin Settings
+
+Project Fortis comes with an admin page with which you can edit site configurations and also provide details on how events should be monitored and processed. Through the admin site, you will be able to configure site settings, blacklist terms, whitelist terms (watchlist), streams (twitter, facebook, etc.), monitored places, and trusted sources.
+
+Once you deployed a site with `deploy to azure`, you may view the admin site by navigating to: http://localhost:8888/#/site/{YOUR_SITE_NAME}/admin
```

Comment:
If the site is deployed to Azure, it won't be accessible via localhost. This should be the URL provided by the FORTIS_ADMIN_INTERFACE_URL_TAG described in the deployment section. https://github.com/CatalystCode/project-fortis-mono#setting-up-a-new-azure-deployment